src/foam/nanos/crunch/predicate/CapabilityDataUpdated.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
We don't already have a predicate which checks for an update?

Reply:
I found only foam.nanos.ruler.predicate.NewEqOld which checks whole object

Reply:
So despite the above, I have a comment on this.
What we need here, is a way to know when a ucj data is updated on purpose and when a ucj data is just being typed in by user in the wizard, which is always doing updates to save the data.

Generally thats why we have the status of UCJ's .... so in general this should be unnecessary predicate.. however ... the riskRating which is what this is for is going to be a little different cause we grant that immediately.

thinking we add some model level validation to RiskRating, then the rule for IDM can rely on the status of the ucj...if that changes to granted...